db/migrate/20230921011451_add_tweets_arrayt_at_hashtags.rb

```diff
@@ -0,0 +1,5 @@
+class AddTweetsArraytAtHashtags < ActiveRecord::Migration[7.0]
+  def change
+    add_column :hashtags, :tweets, :string, array: true
```

Comment:
Why are you defining this arrays field? it seems this option is only available for PostgreSQL, by adding it you are making your active record schema dependent of the database why are you adding this change?

Reply:
I followed the basic structure you gave us for the table Hashtags in the ERD 🧐 I thought it was meant to store an array of hashtags, but I guess that's not how I should define it.